manifests/config.pp

```diff
@@ -168,8 +168,14 @@
         ssl_content => template('foreman/lookup_identity.conf.erb'),
       }
 
-      foreman::config::apache::fragment { 'auth_kerb':
-        ssl_content => template('foreman/auth_kerb.conf.erb'),
+      if ($facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '8') >= 0) {
+        foreman::config::apache::fragment { 'auth_gssapi':
+          ssl_content => template('foreman/auth_gssapi.conf.erb'),
+        }
```

Comment:
Rather than including a fragment, should we use a directory entry? It was added in https://github.com/puppetlabs/puppetlabs-apache/commit/891fdaf24992e0e96b0ae14e2eca8f1991db373a which means we should also raise the minimum version to 5.7.0 in metadata.json, but that's ok IMHO.

Reply:
Maybe. Probably.

Right now it's just copy paste to see if it even works :D

Reply:
Right now, I'd prefer to skip this, as that would require more changes: when you specify `directories`, you also need to pass all other directories you want to configure, like the one for the docroot, as specifying it makes the defaults be dropped.